app/presenters/menu/default_menu.rb

```diff
@@ -70,18 +70,18 @@ def container_menu_section
         Menu::Section.new(:cnt, N_("Containers"), [
           Menu::Item.new('ems_container',     N_('Providers'),     'ems_container',     {:feature => 'ems_container_show_list'},     '/ems_container'),
           Menu::Item.new('container_project', ui_lookup(:tables => 'container_project'), 'container_project', {:feature => 'container_project_show_list'}, '/container_project'),
-          Menu::Item.new('container_node',    ui_lookup(:tables => 'container_node'),    'container_node',    {:feature => 'container_node_show_list'},    '/container_node'),
+          Menu::Item.new('container_node',    N_('Container Nodes'),    'container_node',    {:feature => 'container_node_show_list'},    '/container_node'),
           Menu::Item.new('container_group',   ui_lookup(:tables => 'container_group'),   'container_group',   {:feature => 'container_group_show_list'},   '/container_group'),
           Menu::Item.new('container_route',   ui_lookup(:tables => 'container_route'),   'container_route',   {:feature => 'container_route_show_list'},   '/container_route'),
           Menu::Item.new('container_replicator', ui_lookup(:tables => 'container_replicator'),   'container_replicator',   {:feature => 'container_replicator_show_list'},   '/container_replicator'),
-          Menu::Item.new('container_service', ui_lookup(:tables => 'container_service'), 'container_service', {:feature => 'container_service_show_list'}, '/container_service'),
+          Menu::Item.new('container_service', N_('Container Services'), 'container_service', {:feature => 'container_service_show_list'}, '/container_service'),
           Menu::Item.new('container',
                          ui_lookup(:tables => 'container'),
                          'containers',
                          {:feature => 'containers', :any => true},
                          '/container/explorer'),
           Menu::Item.new('container_image',
-                         ui_lookup(:tables => 'container_image'),
+                         N_('Container Images'),
```

Comment:
> Should all of these be in their own section in en.yml instead of manually setting them with N_()?
> @abonas @simon3z

please ask @martinpovolny , he is working on guidelines for this.